megamek/src/megamek/client/ui/swing/minimap/Minimap.java

```diff
@@ -707,6 +744,19 @@ private void drawMap(boolean forceDraw) {
         }
     }
 
+    private void paintMoveTracks(Graphics g) {
+        Graphics2D g2d = (Graphics2D) g.create();
+        Stroke dashed = new BasicStroke(3, BasicStroke.CAP_BUTT, BasicStroke.JOIN_BEVEL,
+            0, new float[]{6}, 3);
+        g2d.setStroke(dashed);
+        Color previousColor = g.getColor();
```

Comment:
With a new g2d created, I think the previous color doesnt need to be restored (?)